a10_octavia/common/openstack_mappings.py

```diff
@@ -18,8 +18,8 @@ def hm_type(c, os_hm_type):
         'PING': c.slb.hm.ICMP,
         'TCP': c.slb.hm.TCP,
         'HTTP': c.slb.hm.HTTP,
-        'HTTPS': c.slb.hm.HTTPS,
-        'UDP' : c.slb.hm.UDP
+        'HTTPS': c.slb.hm.HTTPS
+        # TODO: Support for UDP Health Monitor (at acos-client)
```

Comment:
before making this change please make sure it is not getting referred somewhere and it might break something

Reply:
Ok, If UDP health mons are supported, reverting this change

Reply:
Now, added support for UDP-CONNECT health monitors